kube-flux/kube-flux
# Create a CRUD (create, read, update, delete) APIs for `policy` objects

- [x] Set up Server

- [x] Add Makefile

- [x] Add initialization

- [x] Add GET

- [x] Add PUT

How to test it?
Under `kube-flux/policy` package, run:
`make zeus`
to build the executable.
Then run:
`./build/zeus`
to start the server.
Then, in another command-line window, run the commands:
`curl http://localhost:9999/`
to GET the policy, and 
`curl -X PUT -H "Content-Type: application/json" -d '{"Status": "Brown", "UpdatedAt": null}' http://localhost:9999/`
to update the policy with http PUT request. Note that you can change Status & UpdatedAt fields.
